test/e2e/tests/account/add-account.spec.ts

```diff
@@ -13,10 +13,14 @@ import LoginPage from '../../page-objects/pages/login-page';
 import ResetPasswordPage from '../../page-objects/pages/reset-password-page';
 
 describe('Add account', function () {
+  const localNodeOptions = {
+    accounts: 1,
```

Comment:
we pass 1, because in the original test, only the first account was seeded, as it was using the import private key. Now we use the mnemonic, which seeds by default 10 accounts, but in this spec we only want the first account seeded, hence we set this to 1